pyartifactory/models/Auth.py

```diff
@@ -16,3 +16,11 @@ class ApiKeyModel(BaseModel):
 
 class PasswordModel(BaseModel):
     password: SecretStr
+

```

Comment:
It would be great to an enum for all possible scopes, I can see in the [Artifactory api](https://www.jfrog.com/confluence/display/RTF/Artifactory+REST+API#ArtifactoryRESTAPI-CreateToken) that there are 3 possible scopes.
```
class ScopeEnum(str, Enum):
    member_of_groups = "member-of-groups"
    ...
```